### PR TITLE
Translate "Ruby 3.2.0 rc1/preview2/preview3 released" (ko)

### PR DESCRIPTION
**Pull request by riseshia**
:link: #2818

Translation of #2875, #2900, #2913
Actual diff is
- rc1: 8c30c3c
- preview3: 1b8d13d
- preview2: 34121bc

Please review this 🙏

### CODE REVIEW

File: ko/news/_posts/2022-11-11-ruby-3-2-0-preview3-released.md [hunk 1]
```diff
@@ -0,0 +1,395 @@
...
+* RubyVM::AbstractSyntaxTree
+    * `parse`, `parse_file`, `of`에 `error_tolerant` 옵션이 추가되었습니다. [[Feature #19013]]
+
+* Set
+    * Set은 이제 `require "set"`할 필요 없이 사용 가능한 내장 클래스입니다. [[Feature #16989]]
+      현재는 `Set` 상수를 사용하거나 `Enumerable#to_set`을 호출하면 자동으로 로드됩니다.
+
+* String
+    * `String#byteindex`와 `String#byterindex`가 추가되었습니다. [[Feature #13110]]
+    * 유니코드 버전이 15.0.0, 에모지 버전이 15.0으로 갱신되었습니다. [[Feature #18037]]
```
Comment (yous) [inline ko/news/_posts/2022-11-11-ruby-3-2-0-preview3-released.md:219]:
```suggestion
    * 유니코드 버전이 14.0.0, 에모지 버전이 14.0으로 갱신되었습니다. [[Feature #18037]]
```

File: ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md [hunk 2]
```diff
@@ -0,0 +1,491 @@
...
+
+    foo(k: 1)
+    ```
+
+## 성능 향상
+
+### YJIT
+
+* YJIT은 x86-64와 arm64/aarch64 CPU에서 동작하는 Linux, MacOS, BSD, 그 외 UNIX 환경을 지원합니다.
+    * 이번 릴리스는 Apple M1/M2, AWS Graviton, Raspberry Pi 4 등을 추가로 지원합니다.
```
Comment (yous) [inline ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md:208]:
```suggestion
    * 이번 릴리스는 Apple M1/M2, AWS Graviton, Raspberry Pi 4 ARM64 프로세서 등을 추가로 지원합니다.
```

File: ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md [hunk 3]
```diff
@@ -0,0 +1,491 @@
...
+* YJIT을 빌드하기 위해서는 Rust 1.58.0 이상을 요구합니다. [[Feature #18481]]
+    * CRuby를 YJIT과 함께 빌드하기 위해서는 `./configure` 스크립트를 실행하기 전에 1.58.0 이상의 `rustc`를 설치해 주세요.
```
Comment (yous) [inline ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md:210]:
```suggestion
    * CRuby를 YJIT과 함께 빌드하기 위해서는 1.58.0 이상의 `rustc`를 설치한 후 `./configure` 스크립트를 `--enable-yjit`과 함께 실행해주세요.
```

File: ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md [hunk 4]
```diff
@@ -0,0 +1,491 @@
...
+* YJIT은 Object shape를 이용해 최적화합니다. [[Feature #18776]]
+* 상수를 무효화하는 단위를 작게 하여 새 상수를 정의할 때 더 적은 코드를 무효화합니다. [[Feature #18589]]
+
+### MJIT
+
+* MJIT 컴파일러는 표준 라이브러리 `mjit`으로 Ruby를 사용해 재구현되었습니다.
+* MJIT 컴파일러는 MJIT 워커에 의해 실행된 네이티브 스레드 대신
+  포크된 프로세스에서 실행됩니다. [[Feature #18968]]
+    * 이 영향으로 Microsoft Visual Studio(MSWIN)가 더 이상 지원되지 않습니다.
+* MinGW는 더 이상 지원되지 않습니다. [[Feature #18824]]
```
Comment (yous) [inline ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md:232]: Oh, '더 이상' was right. Let's replace '더이상' in 3.2.0 post too.

File: ko/news/_posts/2022-09-09-ruby-3-2-0-preview2-released.md [hunk 5]
```diff
@@ -0,0 +1,329 @@
...
+
+long_time_re =~ "a" * 50000 + "x" # never interrupted
+```
+
+기능 제안 티켓: https://bugs.ruby-lang.org/issues/17837
+
+
+## 그 이외의 주목할 만한 새 기능
+
+### 서드파티 소스 코드를 더이상 내장하지 않음
```
Comment (yous) [inline ko/news/_posts/2022-09-09-ruby-3-2-0-preview2-released.md:75]:
```suggestion
### 서드파티 소스 코드를 더 이상 내장하지 않음
```

File: ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md (hunk 4 above)
Comment (riseshia) [reply ko/news/_posts/2022-12-06-ruby-3-2-0-rc1-released.md:232]: Let's fixe on this pr with 11983930

**Pull request merged**